targets/objc/templates/PlayFabAPI.m.ejs

```diff
@@ -36,29 +36,6 @@ return deviceModel;
     return !([[PlayFab<%- api.name -%>API GetInstance].mUserSessionTicket length]==0);
 }
 
-#ifdef USE_IDFA
```

Comment:
USE_IDFA
Do a global search for this, just in case, this is a new variable that might have further use elsewhere.

Reply:
("New" as in I haven't seen it as part of this changelog until now)